.travis/build-openssl-linux.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ ! -f download-cache/openssl-${OPENSSL_VERSION}.tar.gz ]; then 
+	wget -O download-cache/openssl-${OPENSSL_VERSION}.tar.gz https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz; 
```

Comment:
This will break every time a new OpenSSL version is released (because the referenced version is moved to old/).  Quite annoying, and out of our control. I'd rather not have to update this script for each openssl release.

The only way out I see is doing e.g. `git checkout --single-branch -b OpenSSL_1_0_2-stable', and then `git checkout $(git describe --abbrev=0 --tags)` to get to the most recent release...